content/usgs-packages/geoknife_Assessment.Rmd

```diff
@@ -89,16 +92,19 @@ query(us_meterology, "times")
 
 ## Exercise 3
 
-*What was the average maximum air temperature in Texas on July 4, 2007? Use the dataset titled "TopoWx: Topoclimatic Daily Air Temperature Dataset for the Conterminous United States", which has a maximum temperature variable, tmax. Hint: you will need all three pieces - stencil, fabric, and knife.*
+*What was the average maximum air temperature in Texas on July 4, 2007? Use the dataset titled "TopoWx: Topoclimatic Daily Air Temperature Dataset for the Conterminous United States", which has a maximum temperature variable, tmax. Hint: you will need all three pieces - stencil, fabric, and knife. Also, this dataset uses 12:00 to indicate the start and end of a day.*
```

Comment:
Hmm OK. How about "this data has daily values set at 12:00, and you will need to include times in your start and end dates to include the correct day's value."